README.md

```diff
@@ -62,14 +61,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [N/A](https://google.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] To make your README more professional, I think it would be better if you update your `Getting Started` section with by adding some missed information such as of how to `clone` project,  add the `correct` cloning link, `install` your app, `use/run` your app, `run tests`(if they exist),... **For example to help you, just refer this [article](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)**